docs/docker.md

```diff
@@ -20,6 +20,9 @@ $ docker build -t horovod:latest horovod-docker
 
 After the container is built, run it using [nvidia-docker](https://github.com/NVIDIA/nvidia-docker).
 
+**Note**: you can replace horovod:latest with the [specific](https://hub.docker.com/r/uber/horovod/tags) pre-build 
+docker container with Horovod instead of build it by yourself
```

Comment:
s/build/building/

Reply:
thx for pointing it out. PTAL

Reply:
Looks like force-push did not pick up the changes?

Reply:
done ... yeah, i forgot commit changes before push